src/components/ParticipantStrip/ParticipantStripGrid/ParticipantStripGrid.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Paper from '@material-ui/core/Paper';
+import { useEffect, useState } from 'react';
+import Grid from '@material-ui/core/Grid';
+import useParticipants from '../../../hooks/useParticipants/useParticipants';
+import useVideoContext from '../../../hooks/useVideoContext/useVideoContext';
+import useSelectedParticipant from '../../VideoProvider/useSelectedParticipant/useSelectedParticipant';
+import Participant from '../../Participant/Participant';
+const useStyles = makeStyles(theme => ({
+  root: {
+    flexGrow: 1,
+    display: 'flex',
+    [theme.breakpoints.down('xs')]: {
+      flexWrap: 'wrap',
+    },
+    flexWrap: 'nowrap',
+  },
+  paper: {
+    padding: theme.spacing(1),
+    textAlign: 'center',
+    color: theme.palette.text.secondary,
+  },
+  scrollable: {
+    overflow: 'auto',
+  },
+}));
+
+export interface ParticipantStripGridProps {
+  viewMode: string;
+}
+export default function ParticipantStripGrid({ viewMode }: ParticipantStripGridProps) {
+  const {
+    room: { localParticipant },
+  } = useVideoContext();
+  const [currViewMode, setCurrViewMode] = useState('');
+  const [lgState, setLgState] = useState<any>(4);
+  const [mdState, setMdState] = useState<any>(3);
+  const participants = useParticipants();
+  const [selectedParticipant, setSelectedParticipant] = useSelectedParticipant();
+
+  useEffect(() => {
+    if (currViewMode != viewMode) {
+      if (viewMode.includes('3X3')) {
+        setMdState(4);
+        setLgState(4);
+      } else if (viewMode.includes('4X4')) {
+        setMdState(3);
+        setLgState(3);
+      } else if (viewMode.includes('2X2')) {

```

Comment:
Awesome!!!
Only thing that I'd change is moving the 2x2 to the top of the "if"s list

Reply:
Also, something I think we should consider is hiding these options for small screens, because if a user chooses any grid view on SM screen, he/she actually not perceive a change on the UI

Reply:
so do you want me will do that? is it urgent for this story?

Reply:
Not urgent, but as this is a code review, I wrote what I noticed - maybe open a ticket for that?
I'd say it's in the scope of this ticket, even if it's not specified